modules/storages/app/common/storages/peripherals/storage_interaction/one_drive/copy_template_folder_command.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2024 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module Storages
+  module Peripherals
+    module StorageInteraction
+      module OneDrive
+        class CopyTemplateFolderCommand
+          def self.call(storage:, source_path:, destination_path:)
```

Comment:
I kept the same signature on `.call`. I could rename the params on `#call` so that it makes more sense.

```ruby
def self.call(storage:, source_path:, destination_path:)
  new(storage).call(source_location: source_path, folder_name: destination_path)
end
```

WDYT?

Reply:
I'm fine with either, why I labeled it as yellow :D

personally I'd probably even change `.call` for both, according to the perception, that those commands actually use a "location", not specifically a path.

Reply:
I'll leave `.call` as is as it is the interface. We need to standardize a lot of stuff in a bigger code maintenance task, we can do that later.